package.json

```diff
@@ -3,46 +3,28 @@
   "version": "1.0.0",
```

Comment:
Read http://semver.org/. Typically, the first version of a module is 0.0.1. Once the first release is out, this changes to 0.1.0. After that, every time a feature is added, the middle number (minor version) is incremented. Every time a bug is fixed, the last number (patch version) is incremented. Every time a backwards incompatible change is added, the first number (major version) is incremented. Some modules are go to 1.0.0 once the API is finalized and the module is stable.